etl-job/fhir_import_export.py

```diff
@@ -49,6 +50,54 @@ def _user(auth: Gen3Auth) -> dict:
     return auth.curl('/user/user').json()
 
 
+def load_grip(graph_name, directory_path, output, access_token):
+    """Loads a directory of .ndjson or .gz files to GRIP. TODO: implement schema in grip"""
+    # List graphs and check to see if graph name is amoung the graphs listed
+    response = requests.get("http://local-grip:8201/graphql/list-graphs",
+                        headers= {"Authorization": f"bearer {access_token}"})
+
+    response.raise_for_status()
+    response_json = response.json()
+    assert graph_name in response_json["data"]["graphs"], "ERROR, graph not found in GRIP"
+    output["logs"].append(response)
```

Comment:
maybe append `response_json`? Not sure how the Response object gets stringified

Reply:
good catch